ts/IdentificationModal.tsx

```diff
@@ -92,6 +92,12 @@ const renderIdentificationByBiometryState = (
 const onRequestCloseHandler = () => undefined;
 
 const authenticateConfig: AuthenticateConfig = {
+  title: I18n.t("identification.biometric.popup.title"),
+  sensorDescription: I18n.t("identification.biometric.popup.sensorDescription"),
+  cancelText: I18n.t("global.buttons.cancel"),
+  fallbackLabel: I18n.t("identification.biometric.popup.fallbackLabel"),
+  imageColor: "#0073E6",
```

Comment:
can we use colors from the theme (variables) instead of hardcoding?

Reply:
Done